packages/docs/docs/02-Guides/01-Connecting/09-Tenderly fork.mdx

```diff
@@ -0,0 +1,46 @@
+# Tenderly fork
+
+In some cases, you may want to connect to a network using a different chainId than its default. For example, your application is supposed to support the Mainnet network and forked Mainnet chain using [Tenderly](https://docs.tenderly.co/simulations-and-forks/simulation-api/testing/local-tests-on-top-of-mainnet-data-in-the-cloud/first-usage-of-tenderly-apis-in-tests), so the forked Mainnet should be accessible by a different number than the default chainId of the Mainnet, but the default for both networks it is the same which causes some problems.
+
+## Example
+
+So we may want to do something like this:
+
+```tsx
+const config: Config = {
+  readOnlyChainId: Mainnet.chainId,
+  readOnlyUrls: {
+    [Mainnet.chainId]: 'https://mainnet.infura.io/v3/[your-infura-key]',
+    1337: 'https://rpc.tenderly.co/fork/[your-mainnet-fork-id]',
+  },
+  multicallAddresses: {
+    1337: Mainnet.multicall2Address ?? '0x5ba1e12693dc8f9c48aad8770482f4739beed696',
```

Comment:
Why `??`? It's either defined in our code or not?

Reply:
It's because `multicall2Address` is an optional field in `Chain` model and its type is incompatible with type `string`, but in this case `Mainnet.multicall2Address as string` would be a better option.